include/examples/simple_limb.h

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file simple_limb.h
+ * @author Zac Gong
+ * @brief A simple LimbBase derived class example, a 1DOF limb with 1 joint at
+ *        the origin and a end effector
+ * @date 2022-09-02
+ *
+ * @copyright Copyright (c) 2022
+ *
+ */
+
+#pragma once
+
+#include <Eigen/Dense>
+#include "kodlab_mjbots_sdk/limb_base.h"
```

Comment:
Insert newline between these includes per [this style guide section](https://drake.mit.edu/styleguide/cppguide.html#Names_and_Order_of_Includes).

Reply:
done